### ISSUE TITLE
NuGet

### ISSUE THREAD

**Issue by gaetandezeiraud** [geoperez/PayPalCore#1]
Hello,
I would like to know if the project would soon be on NuGet? It would be much more convenient and easy to use!

**Comment by geoperez** [comment]
I know, but I was expecting to Paypal to release an official nuget with support to Net Core because this version has some classes commented (like logging), so it's not a full port yet.

**Comment by gaetandezeiraud** [comment]
Ok, thanks ;)

**Comment by bremussolutions** [comment]
I think paypal never provides a SDK for dotnet Core ... Is it possible that you publish your solution as a NuGet package? Even though it is merely a prview or beta version?

By the way, a support for netcoreapp1.1 would be wonderful :-)

**Comment by geoperez** [comment]
Yeah they are not yet supporting net core. Regarding the netcoreapp1.1, you may use this library as net standard 1.6 since netcoreapp is for applications.

**Comment by bremussolutions** [comment]
Because of that PayPal apparently not publish a .Net Core SDK, is it possible to share your library via Nuget? Even if your library not contain the whole functionality of the PayPal SDK. The integrated features could declared in the wiki of this repository.

I am glad to help you, either with documentation or implementation.

Let us publish a own useful SDK which helps all developers to integrate PayPal into .Net Core applications.

**Comment by geoperez** [comment]
They are some issues regarding to ConfigurationManager required to address before to release a full version. Again any PR is welcomed.

**Comment by JeanCollas** [comment]
I think I have fixed some of the issues, however I have no idea if I am still under a github repository

**Comment by geoperez** [comment]
You could fork and clone the repo, then replace the files with your version
and commit them. Then create a Pull request.

**Comment by JeanCollas** [comment]
I have updated it to the latest version of the SDK, feel free to play (I haven't fully tested it yet) :

https://github.com/JeanCollas/PayPal-NET-SDK/tree/Core/Source

The configuration should be initialized by calling the method `ConfigManager.Instance.SetConfig`. (the simplest workaround I have found, even if Dependency Injection would have probably been a better implementation).

The solution is for VS2017, and .Net Std 1.6.

(don't mind the 4.5 project, just unload it, it probably does not compile)

**Comment by JeanCollas** [comment]
By the way, I haven't found how to force TLS 1.2 using .Net Std APIs. 
I read PayPal comments, it will be mandatory, therefore it might be an issue in the future.

**Comment by bremussolutions** [comment]
@JeanCollas What is now the difference between this PayPal SDK port and yours? Is it not wise to work on one project and bundle our competencies?

**Comment by JeanCollas** [comment]
@bremussolutions 
The issue of this project is that files have been extracted out of github and back to a new one, so updates from the original SDK are a bit complicated. I forked the original one and included my and @geoperez changes into the latest version of the SDK (which evolves from time to time, even if I have no idea about the changes because I haven't followed much :) )

The other issue is a "trust" issue: as this repository is not linked to the original one, tracking the changes is quite complex. And when it comes to money, trusting a 3rd party tool is not so easy, so I wanted to show the differences with the original SDK (when I used this project for the first time, I spend 3h checking all changes using a diff tool to ensure there were no tricky things added).

These are the only reasons why I have created a new one, and no intention to work on "my side" on this because I don't have enough time for it ^^ (I would be pleased to add contributors even give the ownership of the repository to somebody willing to work on it if this is possible)

**Comment by geoperez** [comment]
I totally agree with you @JeanCollas. Did you already try to create a Pull request to the official SDK?

**Comment by JeanCollas** [comment]
@geoperez no because it has not been developed to be compatible with the other versions, however little work should be enough to make it compatible ( #region statements to isolate core parts when necessary )

The track change should help anybody willing to work on it ^^
